applications/GeoMechanicsApplication/custom_utilities/transport_equation_utilities.hpp

```diff
@@ -79,6 +86,174 @@ class GeoTransportEquationUtilities
     {
         return -PORE_PRESSURE_SIGN_FACTOR * BiotModulusInverse * outer_prod(rNp, rNp) * IntegrationCoefficient;
     }
-    
+
+    static Matrix CalculateMassMatrixDiffOrder(const Geometry<Node>&          rGeom,
+                                               const Geometry<Node>::Pointer& rpPressureGeometry,
```

Comment:
Would it be possible to have the two geometries as consistent arguments?
```suggestion
                                               const Geometry<Node>& rPressureGeometry,
```

Reply:
Indeed, it is better to provide a reference than a pointer.